Mathlib/Topology/Constructions.lean

```diff
@@ -152,125 +152,125 @@ The topology on this type synonym is inherited without change.
 #align is_open_map_to_dual isOpenMap_toDual
 
-theorem isOpenMap_ofDual : IsOpenMap (ofDual : αᵒᵈ → α) := IsOpenMap.id
+theorem isOpenMap_ofDual : IsOpenMap (ofDual : Xᵒᵈ → X) := IsOpenMap.id
 #align is_open_map_of_dual isOpenMap_ofDual
 
-theorem isClosedMap_toDual : IsClosedMap (toDual : α → αᵒᵈ) := IsClosedMap.id
+theorem isClosedMap_toDual : IsClosedMap (toDual : X → Xᵒᵈ) := IsClosedMap.id
 #align is_closed_map_to_dual isClosedMap_toDual
 
-theorem isClosedMap_ofDual : IsClosedMap (ofDual : αᵒᵈ → α) := IsClosedMap.id
+theorem isClosedMap_ofDual : IsClosedMap (ofDual : Xᵒᵈ → X) := IsClosedMap.id
 #align is_closed_map_of_dual isClosedMap_ofDual
 
-theorem nhds_toDual (a : α) : 𝓝 (toDual a) = map toDual (𝓝 a) := rfl
+theorem nhds_toDual (a : X) : 𝓝 (toDual a) = map toDual (𝓝 a) := rfl
 #align nhds_to_dual nhds_toDual
 
-theorem nhds_ofDual (a : α) : 𝓝 (ofDual a) = map ofDual (𝓝 a) := rfl
+theorem nhds_ofDual (a : X) : 𝓝 (ofDual a) = map ofDual (𝓝 a) := rfl
 #align nhds_of_dual nhds_ofDual
 
 end
 
-theorem Quotient.preimage_mem_nhds [TopologicalSpace α] [s : Setoid α] {V : Set <| Quotient s}
-    {a : α} (hs : V ∈ 𝓝 (Quotient.mk' a)) : Quotient.mk' ⁻¹' V ∈ 𝓝 a :=
+theorem Quotient.preimage_mem_nhds [TopologicalSpace X] [s : Setoid X] {V : Set <| Quotient s}
+    {a : X} (hs : V ∈ 𝓝 (Quotient.mk' a)) : Quotient.mk' ⁻¹' V ∈ 𝓝 a :=
   preimage_nhds_coinduced hs
 #align quotient.preimage_mem_nhds Quotient.preimage_mem_nhds
 
 /-- The image of a dense set under `Quotient.mk'` is a dense set. -/
-theorem Dense.quotient [Setoid α] [TopologicalSpace α] {s : Set α} (H : Dense s) :
+theorem Dense.quotient [Setoid X] [TopologicalSpace X] {s : Set X} (H : Dense s) :
     Dense (Quotient.mk' '' s) :=
   Quotient.surjective_Quotient_mk''.denseRange.dense_image continuous_coinduced_rng H
 #align dense.quotient Dense.quotient
 
 /-- The composition of `Quotient.mk'` and a function with dense range has dense range. -/
-theorem DenseRange.quotient [Setoid α] [TopologicalSpace α] {f : β → α} (hf : DenseRange f) :
+theorem DenseRange.quotient [Setoid X] [TopologicalSpace X] {f : Y → X} (hf : DenseRange f) :
     DenseRange (Quotient.mk' ∘ f) :=
   Quotient.surjective_Quotient_mk''.denseRange.comp hf continuous_coinduced_rng
 #align dense_range.quotient DenseRange.quotient
 
-instance {p : α → Prop} [TopologicalSpace α] [DiscreteTopology α] : DiscreteTopology (Subtype p) :=
+instance {p : X → Prop} [TopologicalSpace X] [DiscreteTopology X] : DiscreteTopology (Subtype p) :=
   ⟨bot_unique fun s _ => ⟨(↑) '' s, isOpen_discrete _, preimage_image_eq _ Subtype.val_injective⟩⟩
 
-instance Sum.discreteTopology [TopologicalSpace α] [TopologicalSpace β] [hα : DiscreteTopology α]
-    [hβ : DiscreteTopology β] : DiscreteTopology (Sum α β) :=
-  ⟨sup_eq_bot_iff.2 <| by simp [hα.eq_bot, hβ.eq_bot]⟩
+instance Sum.discreteTopology [TopologicalSpace X] [TopologicalSpace Y] [h : DiscreteTopology X]
+    [hY : DiscreteTopology Y] : DiscreteTopology (Sum X Y) :=
```

Comment:
```suggestion
    [hY : DiscreteTopology Y] : DiscreteTopology (X ⊕ Y) :=
```

Reply:
I also just pushed a commit doing the same replacement across this file. (Happy to pull into its own PR.)